admin/services/pin-generation-v2.service.js

```diff
@@ -30,12 +30,12 @@ const serviceToExport = {
   /**
    * Return a list of pupils who have an active pin - one that can be used to take a check
    * TODO: add pinEnv
```

Comment:
I think that can be updated, as the ‘isLiveCheck’ param was added...?

Reply:
Good spot! Now removed